test/Box/index.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render, cleanup} from '@testing-library/react';
+import Wrapper from '../helper/wrapper';
+import {ThemeSectionError} from '../../src/ThemeSection';
+import Box from '../../src/Box';
```

Comment:
Could we move the tests right into each individual component's folder? This way everything would be in one place and more visible (so less likely we'll forget to add tests for new components :) )

Do you expect tests to require more than one file? If so we could create a tests folder inside of each component folder.

Reply:
I think we should have 1 file per component. I can move them into the component folder if you prefer.